<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="depthLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth: 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="interpolateButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveXLSXButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>814</x>
      <y>450</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as .xlsx</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="inputTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="saveGraphButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>450</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as .png</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadXLSXButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Import from .xlsx</string>
    </property>
   </widget>
   <widget class="QSlider" name="depthSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QChartView" name="chartView" native="true">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>19</y>
      <width>621</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
